cedar-policy-core/src/parser/grammar.lalrpop

```diff
@@ -94,9 +94,9 @@ pub Policy: Node<Option<cst::Policy>> = {
     <l:@L> <err:!> <r:@R> => { errors.push(err); Node::new(None,l,r) },
 }
 
-// VariableDef := Variable [':' Name] [('in' | '==') Expr]
+// VariableDef := Variable [':' Name] ['is' Add] [('in' | '==') Expr]
```

Comment:
Nit: maybe we should add a comment here saying `Add` shows up here rather than `Name` because we want better error message. Otherwise it could be confusing here because the grammar specifies that a `Name` comes after `is`.